### PR TITLE
[no dps] Misc fixes for docs

### PR DESCRIPTION
**Pull request by oliwiapolec**
# ❗️ Feature branch

Remember to start your branch name with `feature/`, for example, `feature/DPS-2947`. This will generate a preview on labs and allow us to test your changes.

# 🚀 Links
Replace the placeholder data in links 👇

- [Feature branch](https://developers.labs.livechat.com/docs/feature/misc-fixes)

# 📓 Description

Misc fixes for docs including:
- fix some typos
- uniform agent/bot naming for group priority + default group prior in configuration APIs
- add more information about CW Moments

### CODE REVIEW

File: content/management/configuration-api/index.mdx [hunk 1]
```diff
@@ -94,7 +94,7 @@ __*__ Possible values for the `groups[].priority` parameter:
 | `first`        |                                                           The highest chat routing priority. Agents with the `first` priority get chats before others from the same group, e.g. Bots can get chats before regular Agents.                                                           |
```
Comment (oliwiapolec) [inline content/management/configuration-api/index.mdx:94]:
```suggestion
| `first`        |                                                           The highest chat routing priority. Agents with the `first` priority get chats before others from the same group, e.g. Bots can get chats before regular Agents. Corresponds to the **Bot** priority option in the LiveChat App.                                                           |
```

File: content/management/configuration-api/index.mdx [hunk 2]
```diff
@@ -359,7 +359,7 @@ __*__ Possible values for the `groups[].priority` parameter:
...
```
Comment (oliwiapolec) [inline content/management/configuration-api/index.mdx:359]:
```suggestion
| `first`        |                                                           The highest chat routing priority. Agents with the `first` priority get chats before others from the same group, e.g. Bots can get chats before regular Agents. Corresponds to the **Bot** priority option in the LiveChat App.                                                            |
```

File: content/extending-chat-widget/chat-widget-moments/index.mdx [hunk 3]
```diff
@@ -74,7 +74,7 @@ Network tester - app written in React, performing a test of the connection statu
 
 # Moments in LiveChat
 
-Moments can be triggered only by [rich message's](https://www.livechat.com/help/rich-messages/) buttons. Currently, LiveChat supports rich messages delivered by chatbots using our [ChatBot](https://chatbot.com) product or using [MessageBox](/extending-ui/extending-agent-app/agent-app-sdk/#messagebox) integration in Agent App. However, we are still working on other possibilities to provide rich messages to chat, such as using backend Chat APIs.
+Moments can be triggered only by [rich message's](https://www.livechat.com/help/rich-messages/) buttons. Currently, LiveChat supports rich messages delivered by chatbots using our [ChatBot](https://chatbot.com) product, a [MessageBox](/extending-ui/extending-agent-app/agent-app-sdk/#messagebox) integration in Agent App, or by sending a rich message using our [Agent Chat API](/content/messaging/agent-chat-api/). We are still working on other possibilities to provide rich messages to chat, such as using backend Chat APIs.
```
Comment (ale-kacperczyk) [inline content/extending-chat-widget/chat-widget-moments/index.mdx:77]:
```suggestion
Moments can be triggered only by [rich message's](https://www.livechat.com/help/rich-messages/) buttons. Currently, LiveChat supports rich messages delivered by chatbots using our [ChatBot](https://chatbot.com) product, a [MessageBox](/extending-ui/extending-agent-app/agent-app-sdk/#messagebox) integration in Agent App, or by sending a rich message using our [Agent Chat API](/content/messaging/agent-chat-api/).
```

**Review by ale-kacperczyk — COMMENTED** [review]
Did you save the files with changes in tables with formatting?

**Comment by oliwiapolec** [comment]
@ale-kacperczyk I did the "normal" save, I think so :D I'll double-check tho

**Review by ale-kacperczyk — APPROVED** [review]

**Pull request merged**